pkg/test/runtime/runtime.go

```diff
@@ -75,7 +76,7 @@ func BuilderFor(appCtx context.Context, cfg kuma_cp.Config) (*core_runtime.Build
 
 	builder.
 		WithComponentManager(component.NewManager(leader_memory.NewAlwaysLeaderElector())).
-		WithResourceStore(resources_memory.NewStore()).
+		WithResourceStore(store.NewPaginationStore(resources_memory.NewStore())).
```

Comment:
This seems weird and likely wrong as now there's pages on something that shouldn't have pages

Reply:
PaginationStore is not doing pagination unless you pass `store.ListByPage`